pyrobosim_ros_gym/eval.py

```diff
@@ -9,20 +9,47 @@
 """Evaluates a trained RL policy."""
 
 import argparse
-from typing import Dict, List
+from typing import Dict, List, Union
 
-from gymnasium.spaces import Discrete
 import rclpy
+from gymnasium.spaces import Discrete
 from rclpy.node import Node
+from stable_baselines3.common.base_class import BaseAlgorithm
 
-from pyrobosim_ros_gym.envs import get_env_by_name
+from pyrobosim_ros_gym.envs import available_envs_w_subtype, get_env_by_name
 from pyrobosim_ros_gym.policies import model_and_env_type_from_path
 
 
+class ManualPolicy:
+    """A policy that allows manual control of the robot."""
+
+    def __init__(self, action_space):
+        print("Welcome. You are the agent now!")
+        self.action_space = action_space
+
+    def predict(self, observation, deterministic):
+        # print(f"Observation: {observation}")
+        # print(f"Action space: {self.action_space}")
+        possible_actions = list(range(self.action_space.n))
+        while True:
+            try:
+                action = int(input(f"Enter action from {possible_actions}: "))
+                if action in possible_actions:
+                    return action, None
+                else:
+                    raise RuntimeError(f"Action {action} not in {possible_actions}.")
```

Comment:
suggest instead of raising a runtime error (which kills the program, to just re-prompt the user to enter a valid action